core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/SssdConfigEndpoint.java

```diff
@@ -0,0 +1,90 @@
+package com.sequenceiq.cloudbreak.api.endpoint;
+
+import java.util.Set;
+
+import javax.validation.Valid;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+
+import com.sequenceiq.cloudbreak.api.model.IdJson;
+import com.sequenceiq.cloudbreak.api.model.SssdConfigRequest;
+import com.sequenceiq.cloudbreak.api.model.SssdConfigResponse;
+import com.sequenceiq.cloudbreak.doc.ContentType;
+import com.sequenceiq.cloudbreak.doc.ControllerDescription;
+import com.sequenceiq.cloudbreak.doc.Notes;
+import com.sequenceiq.cloudbreak.doc.OperationDescriptions;
+
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+
+@Path("/")
+@Consumes(MediaType.APPLICATION_JSON)
+@Api(value = "/sssd", description = ControllerDescription.SSSDCONFIG_DESCRIPTION)
+public interface SssdConfigEndpoint {
+
+    @POST
+    @Path("user/sssd")
```

Comment:
And just use /user and /account etc. so the final path for every sssd relatwd endpoint  will be something like:

/api/v1/sssd/something